src/components/__tests__/HabitCard.test.tsx

```diff
@@ -13,7 +13,7 @@ it("renders the habit card correctly when a habit hasn't started", () => {
     .create(
       <Provider store={store}>
         <StaticRouter>
-          <HabitCard match={{ params: { id: "0" } }} />
+          <HabitCard id={"0"} collapsed={false} />
```

Comment:
I don't think these need curly braces because they're primitives (I could be wrong!)

```suggestion
          <HabitCard id="0" collapsed=false />
```